test-d/simplify.ts

```diff
@@ -42,45 +42,3 @@ expectType<SomeInterfaceAsTypeWrittenByHand>(valueAsInterface);
 expectAssignable<Record<string, unknown>>(valueAsLiteral);
 expectAssignable<Record<string, unknown>>(valueAsSimplifiedInterface);
 expectNotAssignable<Record<string, unknown>>(valueAsInterface); // Index signature is missing in interface
-
-// Should return the original type if it is not simplifiable, like a function.
-type SomeFunction = (type: string) => string;
-expectType<Simplify<SomeFunction>>((type: string) => type);
-
-class SomeClass {
-	id: string;
-
-	private readonly code: number;
-
-	constructor() {
-		this.id = 'some-class';
-		this.code = 42;
-	}
-
-	someMethod() {
-		return this.code;
-	}
-}
-
-expectType<Simplify<SomeClass>>(new SomeClass());
```

Comment:
Let's keep this, but commented out, and with a TODO comment to fix it in the future and with a reference to the issue that caused it to be commented out.